commands/editmessage.js

```diff
@@ -0,0 +1,58 @@
+const Discord = require("discord.js")
+
+module.exports = async (bot, message, args) => {
```

Comment:
Unless necessary (i.e. to avoid rebinding of this) arrow functions are discouraged for anything more than a quick one-line filter or transform. Avoid arrow functions for anything more than a one-liner.

```suggestion
module.exports = async function (bot, message, args) {
```